### ISSUE TITLE
[Accessibility] Color Contrast Ratios (CCR) are not ideal

### ISSUE THREAD

**Issue by jpmcb** [bottlerocket-os/bottlerocket-project-website#110]
**Path to the page with the problem (_do not_ include domain name):**

Home page

**Describe the problem:**

```
### Summary
Text must exceed CCR threshold

### Definition

Text content must exceed Color Contrast Ratio (CCR) of 3.1 for large and/or bolded text
and 4.5 for any other size or style of text.

### Action

V: Change the foreground and background colors of the 77 text elements to meet the CCR threshold.

### Purpose

The higher the color contrast of text the more easy it is to read,
especially for people with visual impairments.

### Techniques

- Change the foreground color to a more complementary color to the background color.
- Change the background color to a more complementary color to the foreground color.
- Remove background images or verify they do not compromise color contrast requirements.

###  Compliance
WCAG Level AA, Required
WCAG Success Criteria
```

**Screen Shot, if possible**

Screenshots of what I'm seeing that is in the `V` category (should fix asap):

This teal `#6eaca0` background color and `#fffff` white have a CCR of 2.6:

<img width="163" alt="Screen Shot 2023-04-06 at 2 58 54 PM" src="https://user-images.githubusercontent.com/23109390/230492714-363a574a-1752-46ea-847f-4eec8447c45e.png">

<img width="863" alt="Screen Shot 2023-04-06 at 2 59 58 PM" src="https://user-images.githubusercontent.com/23109390/230492900-af6a9efc-c30e-4782-bab8-7303f75690ce.png">

---

This blue and orange is also not ideal: `#ff9636` `#12749d` has a CCR of 2.4

<img width="834" alt="Screen Shot 2023-04-06 at 2 53 18 PM" src="https://user-images.githubusercontent.com/23109390/230491730-bfae9158-a67d-41bd-aa00-eeeed67423b2.png">

---

This orange and white is also setting off alarms: the CCR is 3.4 for `#ffffff` and `#dd6900` but it's recommending at to exceed 4.5.

<img width="889" alt="Screen Shot 2023-04-06 at 2 54 01 PM" src="https://user-images.githubusercontent.com/23109390/230491859-54f58489-ff49-40f6-9f34-3809321e1dc3.png">

---

The footer looks close: CCR of 4.4 for `#ffffff` & `#4b8378` and it's recommending at least 4.5.

<img width="292" alt="Screen Shot 2023-04-06 at 2 54 22 PM" src="https://user-images.githubusercontent.com/23109390/230491917-ed7ca6b9-d309-4d7e-97ef-7b8e6d70c214.png">

---

The header also looks to be firing off on this alert but it might just not be picking up the background color since it things it's `#ffffff` against `#ffffff`

<img width="919" alt="Screen Shot 2023-04-06 at 2 54 39 PM" src="https://user-images.githubusercontent.com/23109390/230491959-55808210-a43f-44c1-bdd2-283f1fc8384b.png">

---

https://www.w3.org/TR/WCAG21/#contrast-minimum
https://www.w3.org/TR/WCAG21/#use-of-color
https://www.w3.org/TR/WCAG21/#contrast-enhanced

**Describe how the site should look?**

Should be as accessible as possible

**Comment by stockholmux** [comment]
Plan of action:
1. Bump text size to 24px (copy) / 48px (heading). This lowers the CCR 3.1 (and it looks better)
(original)
![Screenshot 2023-04-10 at 1 14 50 PM](https://user-images.githubusercontent.com/1152927/230980186-52dfa28b-cb1c-4b89-aca2-90e2fc10dcf2.png)
(new)
![Screenshot 2023-04-10 at 1 14 42 PM](https://user-images.githubusercontent.com/1152927/230980085-d421d698-109a-4f98-b6a2-5332052c08a8.png)

2. "Minimal" section gets a foreground of #0A4158 , bring the CCR to 4.22
3. "Safe Updates" section links go from orange to tan #e4d7d0, which is a CCR of 3.73 (fine for the new/larger typeface)
4. Footer stays the same (4.4 CCR seems close enough for what's practically disclosure text)
